### DISCUSSION TITLE
Browser ignores cache

### DISCUSSION THREAD

**Discussion by lufton** [getgauge/taiko#2627] [General]
So I have strange behavior: if I loaded the page using `openBrowser()`, `goto()` and then do `refresh()` then, the browser ignores the cache and redownloads all resources over again. But if I manually navigate to the same url in the new tab then resources are loaded from the cache.
This is confusing as it looks like the browser is actually saving the cache but don't use it on any page that was opened by Taiko. Please check the video: https://youtu.be/-7lebKLPUik

**Comment by zabil** [comment]
Did you mean the [`reload`](https://docs.taiko.dev/api/reload/) api?
Cache is reloaded by default. Please refer the `ignoreCache` option in the [documentation](https://docs.taiko.dev/api/reload/)

**Reply by lufton** [reply]
Sorry, yes, that was `reload` method, but anyway if you watch the video it will be clear that it is not about it. Cache is not used when I do manual refresh too. But if I open the same page manually it catches the cache and loads much faster.
